### ISSUE TITLE
Slow upload speeds \ direct iscsi san & nfs nas drive

### ISSUE THREAD

**Issue by RM4G00** [projectsend/projectsend#299]
Hi,

Can anyone suggest any tweaks to improve upload speeds?

Thanks

**Comment by lenamtl** [comment]
Upload and download are related to server hardware, internet speed, file size, file quantity.

**Comment by RM4G00** [comment]
slow performance using direct iscsi san & nfs nas drive - 1gb ports ...

**Comment by RM4G00** [comment]
Any advice? tried changing chunk size plupload- via upload from computer file.php - currently trying to workout what's causing the bottleneck

**Comment by sgerhager** [comment]
Same issue here running on a deb8 system with Gbit network, but just a few Mbit up and downstream. Direct downloads from the same apache run perfectly fast. Can not be prob of hardware. Seems to be somewhere in PHP ...

**Comment by loki6000** [comment]
Guys the slow speed upload cannot be due to hard disk and network. Ive installed https://gitlab.com/mojo42/Jirafeau this is a drop file php web thingy. and when i upload a 10mb file its instant. however on project send it take 3 minutes. any updates ?

**Comment by stathiskef** [comment]
I am having slow upload speeds as well on my SSD server, has anyone found the solution? Thanks

**Comment by mjiggidy** [comment]
Yeah, I'm getting ~15 minutes for a 2 GB file upload inside my 10Gb LAN.  With another file upload service on the same server, I get ~2-3 seconds.

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
